### PR TITLE
Adding new scalerel with documentation

### PR DESCRIPTION
**Pull request by mmpagani**
This adds the Thingbaijam et al. (2017) scaling relationship (only interface for the time being).

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,7 @@
+
```
Comment (daniviga) [inline debian/changelog]: Blank line, please remove it.

File: doc/manual/oqum/hazard/00c_msrs.tex [hunk 2]
```diff
@@ -16,14 +16,15 @@ \subsection{Relationships for shallow earthquakes in active tectonic regions}
...
+    \item \cite{Thingbaijam2017} - Define  magnitude scaling relationships for interface. Only the magnitude to rupture-area scaling relationships are implemented here, and are called with the keywords \verb=ThingbaijamInterface=.
 \end{itemize}
 
 \subsection{Magnitude-scaling relationships stable continental regions}
 \begin{itemize}
     \item \cite{ceus2011} - Defines a single magnitude to rupture-area scaling relationship for use in the central and eastern United States: $Area = 10.0^{M_W - 4.336}$. It is called with the keyword \verb=CEUS2011=
 \end{itemize}
 
-\subsection{Miscellaneous Magnitude-Scaling Relationships}
+\subsection{Miscellaneous Magnitude-Scaling Relationships}
```
Comment (daniviga) [inline doc/manual/oqum/hazard/00c_msrs.tex]: Trailing space

File: openquake/hazardlib/tests/scalerel/thingbaijam2017_test.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# import unittest
+
+import os
+import numpy
+from openquake.hazardlib.scalerel.thingbaijam2017 import ThingbaijamInterface
+from openquake.hazardlib.tests.scalerel.msr_test import BaseMSRTestCase
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), 'data/thingbaijam2017')
```
Comment (daniviga) [inline openquake/hazardlib/tests/scalerel/thingbaijam2017_test.py]:
```python
DATA_DIR = os.path.join(os.path.dirname(__file__), 'data', 'thingbaijam2017')
```
to make it cross platform.

File: openquake/hazardlib/scalerel/thingbaijam2017.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+    Implements both magnitude-area and area-magnitude scaling relationships.
+    """
+
+    def get_median_area(self, mag, rake):
+        """
+        Calculates median fault area from magnitude.
+        """
+        return 10**(-3.292+0.949*mag)
+
```
Comment (micheles) [inline openquake/hazardlib/scalerel/thingbaijam2017.py]: PEP8 violations here

File: openquake/hazardlib/scalerel/thingbaijam2017.py [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+        Returns std
+        """
+        return 10**0.150
+
+    def get_median_mag(self, area, rake):
+        """
+        Returns magnitude for a given fault area
+        """
+        return (log10(area)+3.292)/0.949
+
```
Comment (micheles) [inline openquake/hazardlib/scalerel/thingbaijam2017.py]: PEP8 violations here

**Pull request merged**